contracts/src/deployers/HyperdriveDeployerCoordinator.sol

```diff
@@ -323,6 +380,28 @@ abstract contract HyperdriveDeployerCoordinator is
         return _deployments[_deployer][_deploymentId];
     }
 
+    /// @dev Prepares the coordinator for initialization by drawing funds from
+    ///      the LP, if necessary.
+    /// @param _hyperdrive The Hyperdrive instance that is being initialized.
+    /// @param _lp The LP that is initializing the pool.
+    /// @param _contribution The amount of capital to supply. The units of this
+    ///        quantity are either base or vault shares, depending on the value
+    ///        of `_options.asBase`.
+    /// @param _options The options that configure how the initialization is
+    ///        settled.
+    /// @return value The value that should be sent in the initialize
+    ///         transaction.
+    function _prepareInitialize(
+        IHyperdrive _hyperdrive,
+        address _lp,
+        uint256 _contribution,
+        IHyperdrive.Options memory _options
+    ) internal virtual returns (uint256 value);
```

Comment:
is there a reason the return value is named here and unnamed in the implementations?